aem-authoring-toolkit-plugin/src/main/java/com/exadel/aem/toolkit/core/util/PluginReflectionUtility.java

```diff
@@ -188,23 +189,101 @@ public static PluginReflectionUtility fromCodeScope(List<String> elements, Strin
     /**
      * Gets generic list of handler instances invoked from all available derivatives of specified handler {@code Class}.
      * Each is supplied with a reference to {@link PluginRuntimeContext} as required
+     *
      * @param handlerClass {@code Class} object
-     * @param <T> Expected handler type
+     * @param <T>          Expected handler type
      * @return {@link List<T>} of instances
      */
     private <T> List<T> getHandlers(Class<? extends T> handlerClass) {
-        return reflections.getSubTypesOf(handlerClass).stream()
+        List<T> list = reflections.getSubTypesOf(handlerClass).stream()
                 .map(PluginReflectionUtility::getHandlerInstance)
                 .filter(Objects::nonNull)
                 .sorted(Comparator.comparing(handler -> handler.getClass().getCanonicalName())) // to provide stable handlers sequence between runs
                 .collect(Collectors.toList());
+        Graph classGraph = new Graph(list);
```

Comment:
Please consider moving tis code to a separate class. Or perhaps two classes. First in the `Graph` class, second is the utility class that does handlers sorting, whatever handlers are. Or maybe an utility class with `Graph` nested inside (should `Graph` be public anyway?)